axelrod/strategies/oncebitten.py

```diff
@@ -87,9 +87,9 @@ class ForgetfulFoolMeOnce(Player):
     """
 
     name = 'Forgetful Fool Me Once'
-    memoryone = False  # Long memory
+    memory_depth = float('inf')  # Long memory
 
-    def __init__(self, forget_probability=0.1):
+    def __init__(self, forget_probability = 0.05):
```

Comment:
0.1 was arbitrary -- 0.05 seemed like it might be better (more like typical GTFT values?). As far as I know I just made this strategy up, so there's no standard choice.